completions/fio

```diff
@@ -12,15 +12,19 @@ _fio()
         --debug)
             _comp_delimited , -W "process file io mem blktrace verify
                 random parse diskutil job mutex profile time net rate compress
-                steadystate helperthread"
+                steadystate helperthread zbd"
```

Comment:
The list seems to be dynamically available by `--debug help`, though I haven't checked which version of `fio` started to support it.

```bash
$ fio --debug help
fio: dumping debug options:process,file,io,mem,blktrace,verify,random,parse,diskutil,job,mutex,profile,time,net,rate,compress,steadystate,helperthread,zbd,all
$ fio --version
fio-3.29
```

Reply:
Excellent! At least the one in Ubuntu 14 and current Fedora development do, so I think we're good. Done in ce5f74512f952e0f6add9facf023095c85428dd6.